what-is-the-ic/4_consensus/index.md

```diff
@@ -4,9 +4,9 @@ card: /img/what-is-the-ic/consensus.jpg
 cardImageFit: cover
 ---
 
-The Internet Computer Protocol ensures that the nodes any subnet always hold the same canister state—even if multiple nodes of a subnet (up to less than one third) are faulty or misbehave. The IC is neither a proof-of-work, nor a proof-of-stake network, but a so-called *DAO-controlled network*, where the NNS DAO manages subnet node membership.
-The [consensus protocol](/how-it-works/consensus/) has the following desirable properties:
-- Low latency – A small number of rounds of exchange suffice to reach agreement. Normally consensus is reached within 1 to 2 seconds.
-- High throughput – Every consensus execution can handle payloads in the order of megabytes.
-- Cryptographic finality – IC consensus reaches cryptographically-guaranteed finality, that is, finalized state changes cannot be undone.
-- Byzantine fault tolerance (BFT) – Being able to tolerate up to (but less than) one third of arbitrarily faulty nodes is theoretically optimal in the considered partially synchronous communication model.
+The Internet Computer Protocol ensures that the canister state on any subnet is always consistent—even if multiple nodes of a subnet are faulty or misbehave. The IC is neither a proof-of-work, nor a proof-of-stake network, but a so-called DAO-controlled network, where the NNS DAO manages subnet node membership. The [consensus protocol](/how-it-works/consensus/) has the following properties:
+
+* Low latency – A small number of rounds of exchange suffice to reach agreement. Consensus is normally reached within 1 to 2 seconds.
+* High throughput – Every consensus execution can handle payloads in the order of megabytes.
+* Cryptographic finality – IC consensus reaches cryptographically-guaranteed finality, that is, finalized state changes cannot be undone.
+* Byzantine fault tolerance (BFT) – The IC can tolerate up to (but less than) one third of arbitrarily faulty nodes, which is optimal in the considered partially synchronous communication model.
```

Comment:
`which is optimal in the considered partially synchronous communication model.` can this be simplified? I'm sort of knowledgeable about that stuff but still stumped with what that means

Reply:
Obviously, it's a balancing act. We want it to be simple but also correct. If you have a good idea on how to simplify it, let me know!   :-)